pkg/kubernetes/node/v1alpha1/kubernetes_test.go

```diff
@@ -15,54 +15,69 @@
 package v1alpha1
 
 import (
-	"reflect"
 	"testing"
 
-	"github.com/pkg/errors"
+	errors "github.com/openebs/maya/pkg/errors/v1alpha1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	clientset "k8s.io/client-go/kubernetes"
 )
 
-func fakeGetClientsetOk() (cli *clientset.Clientset, err error) {
+func fakeGetClientSetOk() (cli *clientset.Clientset, err error) {
 	return &clientset.Clientset{}, nil
 }
 
-func fakeListfnOk(cli *clientset.Clientset, opts metav1.ListOptions) (*corev1.NodeList, error) {
+func fakeListOk(cli *clientset.Clientset, opts metav1.ListOptions) (*corev1.NodeList, error) {
 	return &corev1.NodeList{}, nil
 }
 
 func fakeListErr(cli *clientset.Clientset, opts metav1.ListOptions) (*corev1.NodeList, error) {
-	return nil, errors.New("some error")
+	return nil, errors.New("fake error")
 }
 
 func fakeGetClientSetNil() (clientset *clientset.Clientset, err error) {
 	return nil, nil
 }
 
 func fakeGetClientSetErr() (clientset *clientset.Clientset, err error) {
-	return nil, errors.New("Some error")
+	return nil, errors.New("fake error")
+}
+
+func fakeGetClientSetForPathOk(fakeConfigPath string) (cli *clientset.Clientset, err error) {
+	return &clientset.Clientset{}, nil
+}
+
+func fakeGetClientSetForPathErr(fakeConfigPath string) (cli *clientset.Clientset, err error) {
+	return nil, errors.New("fake error")
 }
 
 func TestWithDefaultOptions(t *testing.T) {
 	tests := map[string]struct {
-		KubeClient *Kubeclient
+		getClientSet        getClientsetFn
```

Comment:
Can we add these as new test functions than modifying existing ones.

Reply:
I Will implement it

Reply:
Done